UnitedIncome/components
# Input/DropDown Spacing Fixes

### Description
> Please provide a brief description of your changes below.

Makes the necessary spacing fixes on the cards between inputs and dropdowns. I've removed the empty `Valid` string as we're no longer reserving space for error messages. When they activate they will make a noticable content jump.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Look at the card.
* QuestionCard is a good one to look at.

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
